administration/troubleshooting.md

```diff
@@ -113,25 +115,25 @@ Fluent Bit v2.1.8
 [0] dummy.0: [[1689971342.068613646, {}], {"message"=>"dummy"}]
 ```
 
-With that options set the stdout plugin is now emitting traces in `json_lines` format:
+With that option set, the stdout plugin emits traces in `json_lines` format:
 
 ```json
 {"date":1689971340.068745,"type":1,"trace_id":"0","plugin_instance":"dummy.0","records":[{"timestamp":1689971340,"record":{"message":"dummy"}}],"start_time":1689971340,"end_time":1689971340}
 ```
 
-All three options can also be defined using the much more flexible `--trace` option:
+All three options can also be defined using the more flexible `--trace` option:
 
 ```bash
-$ fluent-bit -Z -i dummy -o stdout -f 1 --trace="input=dummy.0 output=stdout output.format=json_lines"
+fluent-bit -Z -i dummy -o stdout -f 1 --trace="input=dummy.0 output=stdout output.format=json_lines"
 ```
 
-We defined the entire tap pipeline using this configuration: `input=dummy.0 output=stdout output.format=json_lines` which defines the following:
+This example defines the tap pipeline using this configuration: `input=dummy.0 output=stdout output.format=json_lines` which defines the following:
 
-  * input: dummy.0 (listens to the tag and/or alias `dummy.0`)
-  * output: stdout (outputs to a stdout plugin)
-  * output.format: json_lines (sets the stdout format o `json_lines`)
+- `input`: `dummy.0` - listens to the tag or alias `dummy.0`
+- `output`: `stdout` - outputs to a stdout plugin
+- `output.format`: `json_lines` - sets the stdout format to `json_lines`
```

Comment:
```suggestion
- `input`: `dummy.0` listens to the tag or alias `dummy.0`.
- `output`: `stdout` outputs to a stdout plugin.
- `output.format`: `json_lines` sets the stdout format to `json_lines`.
```

Reply:
yeah i debated wht do do here because I wasn't sure that the two code fences would not be confusing but I'm also 🤷🏼